### DISCUSSION TITLE
Need Guide on how to use Async Get Bytes for tracking in Progress Bar

### DISCUSSION THREAD

**Discussion by studio-suman** [omansak/libvideo#224] [General]
@omansak - Hello, Hope you are doing well and Happy New Year Wishes in advance, need guidance on how to show in progress bar.

Below is a snippet of the code
                    var yt = YouTube.Default;
                    var video = await yt.GetVideoAsync(textBox1.Text);
                    File.WriteAllBytes(fbd.SelectedPath+@"\"+video.FullName, await video.GetBytesAsync());

Pls help to use progress bar here, I am newbie here.

**Comment by omansak** [comment]
Thank you for feedback, Happy new year

you can look it , https://github.com/omansak/libvideo/blob/master/docs/README.md

**Reply by studio-suman** [reply]
Thank you, I have gone through the same!!

**Reply by studio-suman** [reply]
Still Not able to do without using HTTP Client
